money-core/src/main/resources/reference.conf

```diff
@@ -54,6 +54,21 @@ money {
         configuration = {
           emitter = "com.comcast.money.emitters.LogEmitter"
           log-level = "WARN"
+          span-start = "Span: "
+          null-value = "NULL"
```

Comment:
does it make sense for these to be under the emitter configuration?

maybe something like:
emitter = {
     class="com.comcast.money.emitters.LogEmitter
     formatting = {
          span-start = "Span: "
          ...
    }
}